PyStemmusScope/stemmus_scope.py

```diff
@@ -12,32 +13,95 @@
 
 logger = logging.getLogger(__name__)
 
+def _is_model_src_exe(model_src_path: Path):
+    #TODO add docstring
```

Comment:
Do you still want to add the docstring for this PR?

Reply:
Added.